lib/elixir/lib/kernel.ex

```diff
@@ -823,6 +823,10 @@ defmodule Kernel do
   @doc """
   Returns the tail of a list. Raises `ArgumentError` if the list is empty.
 
+  If a one-element list is passed, it returns an empty list.
```

Comment:
I don't think we need this clarification. The tail of a one-element list is an empty list.